typescriptSrc/animator.ts

```diff
@@ -4,6 +4,7 @@
 /// <reference path="assert.ts" />
 /// <reference path="backtracking.ts" />
 /// <reference path="collections.ts" />
+/// <reference path="createHTMLElements.ts" />
```

Comment:
This is too for some even though this is a comment. I don't really understand why that is the case.